### ISSUE TITLE
The dimension of sequence logist 

### ISSUE THREAD

**Issue by wwzll123** [evolutionaryscale/esm#86]
First of all thank you for your great work in integrating protein sequence, structure and function into a single model!

I found that the shape of output of sequence regression_head is L×64, which confused me. The size of the sequence vocab defined in constants\esm3.py is 33. Intuitively, the output dimensions of the sequence logistic should be the same as the sequence vocab. Just like in ESM2. According to your description in the pre-print, this output of logistic regression should be used directly to calculate the cross-entropy with the primary sequence. This is obviously strange. 
![2de5ce1b05e3e86a041bb4e06b65824](https://github.com/user-attachments/assets/80540a55-811e-463e-9e66-e11b5c974103)

In addition, I found the "Xseq" is missing in the section "A.1.4. ESM3 Inputs and Forward Pass".
![1724074100243](https://github.com/user-attachments/assets/54195f19-ccdf-4d74-88ac-288247dd9b5a)

**Comment by ebetica** [comment]
We use 64 to improve model throughput. Only 29 tokens are used (section A.1.3 for description). Thanks for pointing out the error in the paper!

**Comment by kentwait** [comment]
I would like to explicitly clarify how to read the 64-dim sequence logits output in the `LogitsOutput` object.

I am porting some code from ESM2 to work with ESM3/ESMC.
Looking at the ESM3 code, `SEQUENCE_VOCAB` in `esm/esm/utils/constants/esm3.py` has 33 items in its list - the same set and order of tokens in ESM2.
```
SEQUENCE_VOCAB = [
    "<cls>", "<pad>", "<eos>", "<unk>",
    "L", "A", "G", "V", "S", "E", "R", "T", "I", "D", "P", "K",
    "Q", "N", "F", "Y", "M", "H", "W", "C", "X", "B", "U", "Z",
    "O", ".", "-", "|",
    "<mask>",
]
```
For example, in ESM2 index 0 of the logits tensor corresponds to the logic for the <cls> (BOS) token. Or index 5 for L, etc.

As mentioned in section A.1.3

> Protein sequences are tokenized as the 20 canonical amino acids, plus BOS, EOS, mask, pad, unknown. 
We keep four non-standard amino acids as in Lin et al. (5), B - Asparagine, U - selenocysteine, Z - glutamic acid, and O - ornithine. This totals to 29 tokens.

Does this mean that indices 0-23, 25-28, and 32  in the 64-dim tensor correspond to `["<cls>", "<pad>", "<eos>", "<unk>", "L", "A", "G", "V", "S", "E", "R", "T", "I", "D", "P", "K", "Q", "N", "F", "Y", "M", "H", "W", "C", "B", "U", "Z", "O", "<mask>"]`? Or conversely, indices 24, 29-31 and 33-63 of the 64-dim tensor should be ignored? 

Or is there a different ordering such that only the first 29 of the 64 values are meaningful?
